src/Tribe/REST/V1/Endpoints/Archive_Event.php

```diff
@@ -434,7 +434,7 @@ public function get_documentation() {
 	 *
 	 * @return array
 	 */
-	public function GET_args() {
+	public function READ_args() {
```

Comment:
Flagging this since we're removing a public method without deprecating it (there some similar changes in the matching Common PR) ... in this scenario it's probably not going to cause any problems for anyone, felt it was worth highlighting though.